docusaurus/docs/reactnative/core-components/channel.mdx

```diff
@@ -702,6 +705,10 @@ Callback function to set the [ref](https://reactjs.org/docs/refs-and-the-dom.htm
 
 <DateHeader />
 
+### EditingStateHeader
```

Comment:
To make more explicit, I think we can put `Input` prefix

- InputEditingStateHeader
- InputReplyStateHeader
- InputGiphySearch

Reply:
Done